ci/scripts/tests/workflow.xml

```diff
@@ -0,0 +1,284 @@
+<?xml version="1.0"?>
+<!DOCTYPE workflow
+[
+	<!--
+	PROGRAM
+		Main workflow manager for Global Forecast System
+
+	NOTES:
+		This workflow was automatically generated at 2024-05-01 14:24:16.515689
+	-->
+	<!ENTITY PSLOT "C48_ATM_0ad155f1">
+	<!ENTITY ROTDIR "/work2/noaa/stmp/CI/ORION/2555/RUNTESTS/COMROOT/C48_ATM_0ad155f1">
```

Comment:
This is not going to be static, another reason why this file does not belong in the repo.